ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/cli/Cli.scala

```diff
@@ -137,7 +137,7 @@ object Cli {
             .fold(Some(TlsConfiguration(enabled = true, None, None, None, clientAuth)))(c =>
               Some(c.copy(clientAuth = clientAuth)))))
 
-      opt[Int]("maxInboundMessageSize")
+      opt[Int]("max-inbound-message-size")
```

Comment:
This is a breaking change. Can we support both and only emit a warning for the old syntax?

Reply:
Yes, fixed that. What's our process for deprecating command-line parameters?